ambry-store/src/main/java/com.github.ambry.store/DiskManager.java

```diff
@@ -45,39 +46,46 @@
   private final Time time;
   private final DiskIOScheduler diskIOScheduler;
   private final ScheduledExecutorService longLivedTaskScheduler;
+  private final DiskSpaceAllocator diskSpaceAllocator;
   private final CompactionManager compactionManager;
+  private volatile boolean running = false;
```

Comment:
Removed the volatile. This would only be an issue if getStore was still being called when shutdown is called

This is needed because we want to disable all stores if the DSA fails to start up.